src/cbc_sdk/platform/asset_groups.py

```diff
@@ -75,34 +95,41 @@ def _query_implementation(cls, cb, **kwargs):
         return AssetGroupQuery(cls, cb)
 
     @classmethod
-    def create_group(cls, cb, name, description, policy_id):
+    def create_group(cls, cb, name, description, policy_id, query=None):
         """
         Create a new asset group.
 
         Required Permissions:
-            gm.group-set (CREATE)
+            gm.group-set(CREATE)
 
         Args:
             cb (BaseAPI): Reference to API object used to communicate with the server.
             name (str): Name for the new asset group.
             description (str): Description for the new asset group.
             policy_id (int): ID of the policy to be associated with this asset group.
+            query (str): Query string to be used to dynamically populate this group. Default is ``None``.
 
         Returns:
             AssetGroup: The new asset group.
         """
-        group_data = {"name": name, "description": description, "member_type": "DEVICE", "policy_id": policy_id}
+        group_data = {"name": name, "description": description, "member_type": "DEVICE",
```

Comment:
Policy_id and query are both optional.
If query is null, it needs to be excluded from the request.
This is a valid create group request body:
{
    "description": "Testing new SDK",
    "member_type": "DEVICE",
    "name": "SDK Test 01"
}